cpp/src/sampling/neighbor_sampling_impl.hpp

```diff
@@ -75,11 +77,29 @@ neighbor_sample_impl(
 #else
```

Comment:
The #ifdef NO_CUGRAPH_OPS should be deleted.  I must have missed this in my PR that moved the sampling function into cugraph.